webpack/assets/javascripts/react_app/stores/NotificationsStore.js

```diff
@@ -12,9 +12,13 @@ class NotificationsEventEmitter extends AppEventEmitter {
   }
 
   getNotifications() {
-    return (_notifications.data || []);
+    return (_notifications.data || {});
   }
 
+  hasUnreadNotifications(notifications) {
+    return !!(notifications.find(notification => notification.seen));
+
+  }
```

Comment:
function does not return correct result

I suggest checking `_notifications` (which is an object and does not support `find`) instead of calling the method with an argument